boards/DCM/Src/App/App_SetPeriodicCanSignals.c

```diff
@@ -0,0 +1,65 @@
+#include "App_SetPeriodicCanSignals.h"
+#include "App_DcmWorld.h"
+#include "App_CanRx.h"
+#include "App_CanTx.h"
+
+// Regen allowed when braking or (speed > 5kmh and AIRs closed)
+static bool RegenAllowed(struct DcmCanRxInterface *can_rx)
```

Comment:
This function is used once. You should move the following code into the fxn below

Reply:
Is this to conserve memory? I just figured it improves readability

Reply:
Nope in this case I feel like the code for this static function is pretty simple. I can still understand what you're trying to do if I see the following lines of code inside of the SetPeriodicSignals function.

Reply:
alright, will remove